logstash-core/lib/logstash/config/defaults.rb

```diff
@@ -6,6 +6,14 @@ module LogStash module Config module Defaults
 
   extend self
 
+  def input
```

Comment:
I understand, still thinks constants are better suited for this kind of data. The difference for me is this are string while the other is coming from a method call. I think we should move to constant as it was previously. But I agree is good to have this methods in here.